### DISCUSSION TITLE
Enhancement ideas

### DISCUSSION THREAD

**Discussion by AseKarlsson** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#425] [Ideas]
- I only use cleaning by “Rooms” at the moment, maybe the default selection of the default view could be configurable by code (No need to change from “Zone Cleanup” every time).

- I don’t think the vacuum can give this, but maybe your programming skills can. It would be very nice to get a number of days from last clean of a room maybe in overlay name like this ”Living Room (2)” so it still keeps short.

Thanks,
Ase

**Comment by PiotrMachowski** [comment]
Thanks for your suggestions :)
- Default view is just a first view. To make rooms default you have to reorder values in `map_modes`
- Such detection is already possible to achieve using attributes of the camera and an automation. Do you want to display it as a room name over the map?

**Comment by AseKarlsson** [comment]
- Will take a look at map_modes and try to solve it, SOLVED
- Idea was as you discribe, an overlay, could be togeteher with the name I already show, now "Living Room", could like this "Living Room (2)" (meaning last clean was 2 days ago). Here I need more help with the automation part, any good examples?
